app/src/main/java/org/sopt/and/LogInActivity.kt

```diff
@@ -0,0 +1,22 @@
+package org.sopt.and
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import org.sopt.and.screen.LogInScreen
+import org.sopt.and.ui.theme.ANDANDROIDTheme
+
+class LogInActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+
+
+        super.onCreate(savedInstanceState)
+        setContent {
+            ANDANDROIDTheme {
+                    LogInScreen()
+            }
```

Comment:
```suggestion
            ANDANDROIDTheme {
                LogInScreen()
            }
```

인덴트는 4칸 or 2칸 정도로 줄여보시는걸 추천드립니다